### PR TITLE
MES-3833: Displaying overlay when Reverse left is tapped

### PR DESCRIPTION
**Pull request by szabikr**
## Description

Displaying overlay when Reverse left button is pressed.

## Checklist

- [x] PR link added to JIRA ticket
- [x] PR title includes the JIRA ticket number
- [x] Code has been tested manually
- [x] Branch is rebased against the latest develop
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
IMG_1E32E1471AC0-1

### CODE REVIEW

File: src/pages/test-report/cat-be/components/reverse-left-popover/reverse-left-popover.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+reverse-left-popover {
+  background-color: #f9f9f9;
```
Comment (TomBillingtonUK) [inline src/pages/test-report/cat-be/components/reverse-left-popover/reverse-left-popover.scss]: is this in the colors array?

**Review by TomBillingtonUK — APPROVED** [review]
Awesome work - someones having a great coding day ;)

File: src/pages/test-report/cat-be/components/reverse-left-popover/reverse-left-popover.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          [manoeuvre]="manoeuvreTypes.reverseLeft"
+          [competency]="competencies.controlFault"
+          [attr.id]="getId(competencies.controlFault)"
+        ></manoeuvre-competency>
+        <manoeuvre-competency
+          class="second"
+          [attr.id]="getId(competencies.observationFault)"
+          [manoeuvre]="manoeuvreTypes.reverseLeft"
+          [competency]="competencies.observationFault"
+          ></manoeuvre-competency>
```
Comment (rossfauldsbjss) [inline src/pages/test-report/cat-be/components/reverse-left-popover/reverse-left-popover.html]: Could we have the attributes in the same order please, just makes it easier to compare. (trivial i know).
Also is the class second being used to apply styling/positioning?

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/pages/test-report/cat-be/components/reverse-left-popover/reverse-left-popover.scss (hunk 1 above)
Comment (szabikr) [reply src/pages/test-report/cat-be/components/reverse-left-popover/reverse-left-popover.scss]: Added them to `$colors` array

File: src/pages/test-report/cat-be/components/reverse-left-popover/reverse-left-popover.html (hunk 2 above)
Comment (szabikr) [reply src/pages/test-report/cat-be/components/reverse-left-popover/reverse-left-popover.html]: Changed the order and deleted that class

**Pull request merged**